OneSignalSDK/onesignal/in-app-messages/src/main/java/com/onesignal/inAppMessages/internal/display/impl/InAppMessageView.kt

```diff
@@ -626,11 +641,17 @@ internal class InAppMessageView(
                 }
             }
 
+        var overlayColor = Color.parseColor("#BB000000")
+
```

Comment:
Since this block of code repeats twice, we can make it into a method and call that method when we need to get the color.

`Color.parseColor("#00000000")` below is also the transparent color. We can consolidate into one variable and prefer `Color.TRANSPARENT` for clarity.